ros/src/twist_controller/twist_controller.py

```diff
@@ -13,45 +13,60 @@ def __init__(self, steer_ratio, decel_limit, accel_limit):
         self.decel_limit = decel_limit
         self.accel_limit = accel_limit
 
-        self.throttle_pid = PID(1, 0.1, 0.1, self.decel_limit, self.accel_limit)
-        self.brake_pid = PID(1, 0.1, 0.1, self.decel_limit, self.accel_limit)
+        self.throttle_pid = PID(.15, .00, 0, self.decel_limit, self.accel_limit)
+        self.brake_pid = PID(15, 1, 0, 0, 100)
```

Comment:
How did you decide these parameters?

Reply:
I chose those parameters through trial and error. By using my plotting script, I was able to see the individual contribution of the proportional error and integrator (I did not use the derivative).

Reply:
The braking PID controller, has no feedforward, which means we rely on the PID to do everything. For a speed error, of -1mps, if my proportional gain is 15, then PID controller will command 15 Nm of braking torque + the contribution of the integrator.

Reply:
Looks great, looking forward to visualizing all the data!